SpockOpaleye/src/UserApi.hs

```diff
@@ -17,49 +17,46 @@ module UserApi
   , deactivateUser
   ) where
 
-import           ApiBase
+import           AppCore
 import           Control.Arrow
 import           Control.Lens
 import           Control.Monad.IO.Class
 import           Data.Maybe
-import           DataTypes
 import           GHC.Int
 import           Opaleye
-import           OpaleyeDef
 
-import           CryptoDef
 import           Prelude                    hiding (id)
 
-createUser :: UserIncoming -> AppM (Auditable User)
+createUser :: UserIncoming -> AppM User
 createUser user = do
   Just hash <- liftIO $ bcryptPassword $ user ^. password
   let fullUser = user { _userpolyPassword = hash }
-  auditable <$> (createRow userTable fullUser)
+  createRow userTable fullUser
 
-updateUser :: Auditable User -> AppM (Auditable User)
+updateUser :: User -> AppM User
 updateUser user = updateAuditableRow userTable user
```

Comment:
Shouldn't compile as we are unwrapping it in these functions to get the underlying type before calling opaleye's functions on it.